frontend/src/routes/Routes.tsx

```diff
@@ -57,6 +58,10 @@ export const AppRoutes = () => {
         element={<IssuanceErrorPage />}
       />
       <Route path="*" element={<UniversalUnexpected />} />
+      <Route
```

Comment:
Put this above the universal unexpected route, and below the issuance error route